drivers/SmartThings/zwave-switch/src/test/test_qubino_temperature_sensor_with_power.lua

```diff
@@ -15,7 +15,6 @@
 local test = require "integration_test"
 local t_utils = require "integration_test.utils"
 local capabilities = require "st.capabilities"
-local constants = require "st.zwave.constants"
```

Comment:
Assuming this is totally harmless but just wanted to ask about it since it seems to be totally unrelated.

Reply:
changed something in the `drivers/` path to trigger the tests running

also it's an unused import so why not